src/consumer/frame.js

```diff
@@ -134,6 +135,15 @@ class Frame extends EventEmitter {
       iframe.style.overflow = 'hidden';
       iframe.scrolling = 'no';
     }
+
+    // Put custom attributes on iframe if specified
+    Object.entries(this.iframeAttrs).forEach(([key, value]) => {
+      iframe[key] = value;
```

Comment:
I feel like this should use the [setAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute) API.

Also NIT: you can just do `Object.entries(this.iframeAttrs).forEach(([key, value]) => iframe.setAttribute(key, value));

Reply:
Curious about the difference btw setAttribute and accessing attribute directly. Can you elaborate on that  usage a bit?
For the NIT suggestions, ESLinter will complaint about it as forEach's function doesn't expect a return value. That's why I use curly braces for it.

Reply:
> Curious about the difference btw setAttribute and accessing attribute directly. Can you elaborate on that usage a bit?
I read that basically for 1st class properties it doesn't matter, but if you expect to assign non-standard attributes then use the DOM API. I think from that guidance we should use the DOM API to ensure that custom non-standard attributes are supported for whatever reason.